src/OrchardCore/OrchardCore.DisplayManagement.Liquid/LiquidViewTemplate.cs

```diff
@@ -87,7 +87,8 @@ static LiquidViewTemplate()
             Factory.RegisterBlock<HelperBlock>("block");
             Factory.RegisterBlock<NamedHelperBlock>("a");
             Factory.RegisterBlock<NamedHelperBlock>("zone");
-            Factory.RegisterBlock<NamedHelperBlock>("scriptblock");            
+            Factory.RegisterBlock<NamedHelperBlock>("scriptblock");
+            Factory.RegisterBlock<NamedHelperBlock>("styleblock");
```

Comment:
How about creating new "stylesheet" tag instead of "styleblock" for inline style that support both CSS and SCSS?.

Some of VS code extension already support syntax highlighting for "stylesheet" for liquid syntax. https://shopify.dev/tutorials/develop-theme-use-sections

e.g. https://marketplace.visualstudio.com/items?itemName=sissel.shopify-liquid and https://marketplace.visualstudio.com/items?itemName=killalau.vscode-liquid-snippets